UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Startup.cs

```diff
@@ -145,12 +146,18 @@ public void ConfigureServices(IServiceCollection services)
             )
             .AddHeaderPropagation();
             services.AddScoped<IFileSystemHelper, FileSystemHelper>();
-            services.AddScoped<IFileShareService, FileShareService>();            
+            services.AddScoped<IFileShareService, FileShareService>();
             services.AddScoped<IProductDataService, ProductDataService>();
             services.AddScoped<IProductIdentifierValidator, ProductIdentifierValidator>();
             services.AddScoped<IProductDataProductVersionsValidator, ProductDataProductVersionsValidator>();
             services.AddScoped<IProductDataSinceDateTimeValidator, ProductDataSinceDateTimeValidator>();
             services.AddScoped<IExchangeSetStorageProvider, ExchangeSetStorageProvider>();
+            services.AddScoped<IEventHubLoggingHealthClient, EventHubLoggingHealthClient>();
+
+            services.AddHealthChecks()
+                .AddCheck<FileShareServiceHealthCheck>("FileShareServiceHealthCheck")
+                .AddCheck<SalesCatalogServiceHealthCheck>("SalesCatalogServiceHealthCheck")
```

Comment:
Kindly change name of Catalog to AddCheck<SalesCatalogueServiceHealthCheck>("SalesCatalogueServiceHealthCheck") to maintain consistency of naming convention across project solution.

Reply:
Fixed